src/languages/en.js

```diff
@@ -219,6 +221,16 @@ export default {
             body: 'Welcome to the future of Expensify, your new go-to place for financial collaboration with friends and teammates alike.',
         },
     },
+    thirdPartySignIn: {
```

Comment:
We'll need to duplicate in the 'es.js' file. We can take a stab at translations (Copilot is decent at them) and the ask in `#open-source` for corrections. We can probably wait on that for when we PR to expensify/app.